meinberlin/apps/budgeting/assets/ControlBarDropdown.jsx

```diff
@@ -1,31 +1,33 @@
-import React, { useEffect, useState } from 'react'
+import React, { useState } from 'react'
+
+// FIXME: maybe more elegant way?
+// this is only relevant for the first page load,
+// where the url does not contain all default query params.
+// case 1: if there is a default, get its verbose name
+// case 2: if there is a current, get the name of the current
+// case 3: if there is no default, get the name of the first choice
+const getDefaultName = (f, curr) => {
+  let filterChoice
+  if (curr) {
+    filterChoice = f.choices.filter(c => c[0] === curr)[0]
+  } else if (f.default) {
+    filterChoice = f.choices.filter(c => c[0] === f.default)[0]
+  } else {
+    filterChoice = f.choices[0]
```

Comment:
not sure if i follow, this is to set the wording of the filters. Not the actual filter itself.
If there is no default or current set it will use the first of the choices name. So you suggest to return always a translated 'all' ?